packages/codemods/package.json

```
@@ -1,8 +1,8 @@
 {
   "name": "@fluentui/codemods",
   "version": "0.1.0",
-  "description": "package to enable easy upgrades to new versions",
-  "private": true,
+  "description": "Tool enabling easy upgrades to new versions",
+  "private": false,
```

Shouldn't this specifically say it's for Fluent UI, unless you plan to make it more general? Also we typically just remove the `private` setting.
```suggestion
  "description": "Tool enabling easy upgrades to new Fluent UI versions",
```

---

@joschect - pls fix this